Makefile

```diff
@@ -262,32 +312,31 @@ endif
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-build
 catalog-build: opm ## Build a catalog image.
-	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMG) $(FROM_INDEX_OPT)
 
-# Push the catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
 
-
 ##@ Targets used by CI
 
-.PHONY: check ## Dockerized version of make test
-check:
-	$(DOCKER_GO) "make test"
+.PHONY: check 
+check: ## Dockerized version of make test-no-verify
+	$(DOCKER_GO) "make test-no-verify"
```

Comment:
Do we want to have this kind of test in CI (using `DOCKER_GO`) for SNR and NHC?
@slintes @mshitrit

Reply:
this is not a question for PR review IMHO...

Reply:
IIUC that mean that the UT run in a docker.
I think this can have value when running locally - one example for this is that I can have whatever go version I want installed locally but the tests will still run with the correct version.

Reply:
We have agreed that the use of Dockerized version for running `make test-no-verify` is a good practice for SNR and NHC. It will insure the Go version and eliminate another place to check for Go version (in [post-submit.yaml](https://github.com/medik8s/node-healthcheck-operator/blob/main/.github/workflows/pre-submit.yaml#L20-L23)).